node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+const files = process.argv.slice(2);
+
+files.forEach(file => {
+  fs.readFile(file, 'utf-8', (err, data) => {
+    if (err) {
+      throw err;
+    }
+    console.log(data);
+  });
+});
```

Comment:
Looks like you have a combination of levels 2 and 3 in the same file.
- Level 2 (`read-any.js`) only needs to work with one file.
- Level 3 (`cat.js`) does need to work with multiple files, but it also needs to guarantee that they print out in the same order the user specifies. Since `fs.readFile` is asynchronous, it doesn't guarantee they finish in the same order they start, so you'll need to find some other way to provide that guarantee:

---

![image](https://user-images.githubusercontent.com/16143771/156705757-3acc053a-b8cd-4bf2-910e-e1d0973c276d.png)

Reply:
**Fixed level 2**
<img width="1792" alt="Screen Shot 2022-03-04 at 11 22 58 AM" src="https://user-images.githubusercontent.com/67080126/156830114-40ce49e3-7db0-4f17-b7d3-ff7008fba54c.png">

**Fixed level 3**
https://user-images.githubusercontent.com/67080126/156830245-cec31d4a-a28b-45ce-b7b1-bcfc38f50b5d.mp4